repository/CargoPackageManager.package/CGOOngoingProject.class/instance/addDependency..st

```diff
@@ -1,4 +1,5 @@
 dependencies
 addDependency: aDependency
 
-	dependencies add: aDependency
\ No newline at end of file
+	dependencies add: aDependency.
+	aDependency project: self.
```

Comment:
There is also dependent that is defined on a dependency to know from where come from the dependency.
Could it fit the use case?